AUTHORS

```diff
@@ -485,3 +485,4 @@ zhouhaibing089 <zhouhaibing089@gmail.com>
 六开箱 <lkxed@outlook.com>
 缘生 <i@ysicing.me>
 蒋航 <hang.jiang@daocloud.io>
+Matthew Reidy <matthewreidy5@gmail.com>
```

Comment:
Please keep these lines sorted alphabetically, as an automated tool (run every few months) is usually used to update this list.